src/module/common/component/feedback/Actionable/Actionable.types.ts

```diff
@@ -1,11 +1,11 @@
-import { RowProps } from "@peersyst/react-native-components";
-import { ButtonProps } from "../../input/Button/Button.types";
+import { IconButtonProps, RowProps, SvgIconProps } from "@peersyst/react-native-components";
+import { JSXElementConstructor } from "react";
 
 type ActionablePosition = "left" | "right";
 
 export interface ActionableProps extends RowProps {
+    Action: JSXElementConstructor<SvgIconProps>;
```

Comment:
Podem fer `children: ReactElement` que queda millor. No podem forçar que sigui un icon de cap manera així que crec que aquesta solució és una mica millor

Reply:
Ho deixo com a `action: ReactElement` i esborro la prop de `onAction`